lib/loader.js

```
@@ -275,7 +274,11 @@ function downloadFile(config, manifestEntry) {
       return manifestEntry.hash;
     }
 
-    return getMD5(fileBuffer);
+    if (!fileBuffer) {
```

OK, so you don't have an issue with the fix, just that it's silently passed over. I'll add a warning log in.